packages/block-editor/src/store/actions.js

```diff
@@ -1445,6 +1445,36 @@ export const __unstableSetEditorMode =
 			}
 		}
 
+		// TODO: Cleanup this for better performance/readability.
+		// When switching to zoom-out mode, we need to disable editing mode for all blocks except the sections.
+		const sectionsContainerClientId = select.getSectionsContainerClientId();
+		const sectionsClientIds = select.getClientIdsOfDescendants(
+			sectionsContainerClientId
+		);
+		if ( mode === 'zoom-out' ) {
+			dispatch(
+				setBlockEditingModes( [
+					[ '' /* rootClientId */, 'disabled' ],
```

Comment:
I followed the code paths before, and _I think_ `''` is the only client id that would have been used in the context where the old hook was running. But we should confirm that this is correct.